modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
 }
 ----
 
+[[example5]]
+=== Example: Using ^mediaType and ^mimeType Metadata Selectors
+
+The `^mimeType` selector returns the MIME type of a value without parameters, for example, `application/json`. +
```

Comment:
```suggestion
The `^mimeType` selector returns the MIME type of a value without parameters, for example, `application/json`. 

```

Reply:
I don't think we use the `+` to start a new line like that in our docs.

Another option might be to provide some sort of intro sentence and describe `^mimeType` and `^mediaType` below that sentence in separate bullets.